kinbot/mess.py

```diff
@@ -1,17 +1,27 @@
-from __future__ import division
+from __future__ import print_function
+import logging
 import os
 import subprocess
 import time
 import pkg_resources
-import logging
+
 from kinbot import constants
 from kinbot import frequencies
-from kinbot import cheminfo
+from kinbot.uncertaintyAnalysis import UQ
+
 
 class MESS:
     """
-    Class that reads and write all MESS files and runs MESS
+    Class that read and writes MESS files for UQ analysis
+    Code is the same as mess.py exceppt that it loops to create 'n' files
+    and it randomizes the following parameters within the alloted UQ tolerance
+       1. Stationary point energy (E+ZPE, +/- 0.5 kcal/mol)
+       2. Barrier (E+ZPE, +/- 1.0 kcal/mol)
+       3. Frequencies (cm-1 +/-20%)
+    Ranges were chosen/based on the following paper:  Goldsmith, C. F. PCI, 2013, 177-185
+    'n' = number of input files generated, default = X (will be implemented once code works)
```

Comment:
Default can be 100, but with warning in manual that it's probably just way too small.

Reply:
Okay, I think that's a good idea.